empress_gui.py

```diff
@@ -9,7 +9,10 @@
 import sys
 import pathlib
 
-import matplotlib.pyplot as plt
+import matplotlib
```

Comment:
And remove `import matplotlib` line to as it is already import in `from matplotlib import pyplot as plt`